src/components/schemas/MapsStockResponse.yaml

```diff
@@ -0,0 +1,1253 @@
+                  - - -0.991894174028389
+                    - 45.04023133328951
+                  - - -0.988884275217912
+                    - 45.04581158534811
+                  - - -0.98626098059235
+                    - 45.04576133385034
+                  - - -0.977590770482673
+                    - 45.053310839442744
+                  - - -0.967576417437787
+                    - 45.05772512677965
+                  - - -0.96497129116059
+                    - 45.06206413601458
+                  - - -0.960616210892826
+                    - 45.065410559696815
+                  - - -0.960629588094643
+                    - 45.06559940389966
+                  - - -0.959233121136054
+                    - 45.06564006008419
+                  - - -0.95832071548912
+                    - 45.065510230121006
+                  - - -0.95744451319216
+                    - 45.065532364743646
+                  - - -0.955681927056781
+                    - 45.06543273156997
+                  - - -0.954185853572589
+                    - 45.065503909226585
+                  - - -0.951358043384547
+                    - 45.06581165994418
+                  - - -0.949361981789194
+                    - 45.06582839530593
+                  - - -0.949143098345151
+                    - 45.06597138746374
+                  - - -0.948972418142839
+                    - 45.06589630511472
+                  - - -0.948500365677686
+                    - 45.06587698738471
+                  - - -0.946814569188671
+                    - 45.06614411240745
+                  - - -0.945702964336162
+                    - 45.06643593881787
+                  - - -0.94445239641693
+                    - 45.06656138930063
+                  - - -0.94212005021044
+                    - 45.066139150811395
+                  - - -0.939816904131487
+                    - 45.06595022470315
+                  - - -0.93729749040169
+                    - 45.065218978500255
+                  - - -0.924454340500604
+                    - 45.06057895234887
+                  - - -0.923811080404775
+                    - 45.06047541721529
+                  - - -0.922400495223345
+                    - 45.05958756665301
+                  - - -0.911723509798335
+                    - 45.05733089567757
+        properties:
+          type: object
+          properties:
+            id:
+              type: string
+              example: 4762789835398144
+            name:
+              type: string
+              example: CC Médullienne
+            zoneType:
+              type: string
+              enum:
+                - IRIS
+                - CITY
+                - AGGLOMERATION
+                - DEPARTMENT
+              example: AGGLOMERATION
+            count:
+              type: object
+              properties:
+                count:
+                  type: integer
+                  format: int64
+                  example: 64
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore. -->
:magic_wand: **chore** : Le type "count" est déjà décrit dans le fichier `CountResponse.yaml`, autant l'utiliser en référence d'autant qu'il s'agit bien du même type dans le code de l'API
<!-- decorations: blocking, non-blocking, if-minor, test, security, performance -->
<!-- https://www.24joursdeweb.fr/2021/conventional-comments-faire-des-revues-de-code-avec-le-smiley/ -->